### ISSUE TITLE
Size of evaluator's head

### ISSUE THREAD

**Issue by gfiumara** [biometricITC/Face-Toolbox#5]
In Overview, there are several spots that indicate the size of a picture or mask worn by the evaluator shall be the same size as the target user's head. Does this need to imply that the evaluator's head is also roughly the same size as the target user's head? If the evaluator has a much larger or smaller head, the photo or mask would not be able to be worn in a convincing manner.

**Comment by n-kai** [comment]
I believe that any TOE doesn't take the size of actual enrolled user's head into account during the face recognition so the size of a picture or mask worn by the evaluator shall be the same size as the evaluatr's head, not the target user's head so the evaluator who wear the photo or mask looks naturally from the TOE.
 
Attack 3D Face 1-1 also recommends **"the evaluator may cut off the eye regions of a printed photo, hide behind and exhibit blinking through the holes (Cut photo attack)**". So size of picture should be same as the evaluator's head so that the size of evaluator's eye can fit to the eye region of a printed photo. 

I also find another error. Bold text below should be deleted from Face Toolbox Overview. I will create the new PR after #7 is merged.

3D Artefacts - Worn Photo Face Mask
Artefact creation
The evaluator shall print face images for printed photo attack. **display them on a screen for digital photo attack and replay them on a screen for replay video attack**. Size of face images on artefacts shall be same as the test user’s face.

**Comment by woodbe** [comment]
@n-kai the previous PRs have now all been merged.

**Comment by n-kai** [comment]
Thank you I will create a PR to fix them and more next week.

**Comment by n-kai** [comment]
Sorry I will create a PR by next meeting (I am busy to prepare the ISO meeting starting from this Saturday).
